documentation/oracle/governance/adding_a_price_identifier.md

```diff
@@ -0,0 +1,48 @@
+# Adding a Price Identifier
+
+## What is a price identifier?
+
+A [price identifier](../../synthetic_tokens/glossary.md#price-identifier) is a natural language descriptor of the reference index determining how much collateral is needed for a token sponsor to be properly collateralized. 
```

Comment:
@rcai1 I agree with Matt's early comments: this should be generalized and not specific to synthetic tokens. This first sentence needs to describe the concept of a price identifier as it relates to the DVM. 

Perhaps "price_identifier" shouldn't even be in the glossary in the synthetic token section--the price_identifier is really an oracle concept.

Reply:
Great, how does the following work? Do you have any suggested phrases? 

A price identifier... is a natural language descriptor of a reference index that the oracle will determine upon request.